packages/react-components/src/atoms/Avatar.tsx

```diff
@@ -63,7 +63,7 @@ const circleStyle = css({
   borderRadius: '50%',
 });
 const imageStyle = css({
-  objectFit: 'contain',
+  objectFit: 'cover',
```

Comment:
I get that borders on the pic aren't nice, but showing only half of a face seems *much* worse to me :/

Reply:
Both of them aren't perfect. I spoke with James and we rather have a perfect circle than a perfect avatar, and we can always upload a better picture.